rest-client/src/main/java/org/sagebionetworks/bridge/sdk/rest/ErrorResponseInterceptor.java

```diff
@@ -45,27 +46,12 @@ public Response intercept(Chain chain) throws IOException {
         
         Response response = chain.proceed(request);
         
-        logDeprecationWarning(response);
-        if (response.code() != 200) {
+        if (response.code() > 399) {
```

Comment:
We have no exceptions that are thrown that aren't for 400 or 500 exceptions. 100s are all redirects and stuff...